character-controller.js

```diff
@@ -977,16 +977,28 @@ class UninterpolatedPlayer extends StatePlayer {
       emote: new BiActionInterpolant(() => this.hasAction('emote'), 0, crouchMaxTime),
       movements: new InfiniteActionInterpolant(() => {
         const ioManager = metaversefile.useIoManager();
-        return  ioManager.keys.up || ioManager.keys.down || ioManager.keys.left || ioManager.keys.right;
+        return  ioManager.keys.up || ioManager.keys.down || ioManager.keys.left || ioManager.keys.right || ioManager.keys.space || ioManager.keys.ctrl;
       }, 0),
       movementsTransition: new BiActionInterpolant(() => {
+        const ioManager = metaversefile.useIoManager();
+        return  ioManager.keys.up || ioManager.keys.down || ioManager.keys.left || ioManager.keys.right || ioManager.keys.space || ioManager.keys.ctrl;
+      }, 0, crouchMaxTime),

```

Comment:
is crouchmaxtime right for swimming?

Reply:
`crouchMaxTime` used more like a default value in existing codes.
But it should be better to rename to `defaultMaxTime`.

Reply:
I will change `crouchMaxTime` to `defaultMaxTime`.